docs/howto/request-public-listing.md

```diff
@@ -0,0 +1,78 @@
+
+1. Be transparent about the capabilities and qualities of a charm.
+2. Ensure a common level of quality.
+
+```{caution}
+A listing review is **not** code review. The reviewer may be looking at some of the charm code, and may have comments on it, but the listing review is not a review of the architecture or design of the charm, and is not a line-by-line review of the charm code. Do architecture, design, and code review earlier in the charm development process -- reach out in the [Charm Development](https://matrix.to/#/#charmhub-charmdev:ubuntu.com) Matrix channel if you would like assistance.
+```
+
+## Self-evaluate your charm against the public listing criteria
+
+Carry out your own review of your charm and its metadata. In particular, make sure that:
+
+* The charm does what it is meant to do. If it is straightforward to deploy the charm and use the workload, then ensure there is a tutorial that covers this. If the charm deployment is complex (for example, requires specialised hardware or an entire solution of charms), create a video that demonstrates both deploying the charm and some example use of the workload.
+* The charm's page on Charmhub provides a quality impression. This requires providing suitable metadata in `charmcraft.yaml` and reasonable documentation (more on documentation below).
+* The charm has an icon, in the correct format.
+* Some form of CI workflow exists that automatically releases the charm to an unstable channel on commits to the default branch.
+* The charm has integration tests that run on every change to the default branch, and are passing. At minimum, the tests verify that the charm can be deployed and ends up in a success state, and that the charm can be integrated with at least one example for each 'provides' and 'requires' specified (including optional integrations, but excluding tracing) ending up in a success state. The tests should be run with `charmcraft test`.
+
+With respect to test coverage of the charm, note that:
+
+* Unit tests are recommended, but *not* required.
+* A minimal set of integration tests is required, as outlined in the checklist.
+* There is no minimum for test coverage. We suggest that tests cover at least all configuration options and actions, as well as the observed Juju events, but this is not a requirement for listing.
+* Charms may have additional tests in an external location, particularly if the charm has specific resource requirements (such as specific hardware). If this is the case, please mention it in the review request, ideally providing some mechanism for viewing the tests and their results.
+
+Ensure your charm's documentation focuses on the charm itself. For workload-specific or Juju-related content, link to the appropriate upstream documentation. A smaller charm can have single-page documentation for its description. A bigger charm should include a full [Diátaxis](https://diataxis.fr) navigation tree. Ensure that the charm has documentation that covers:
```

Comment:
Documentation is first raised in the first paragraph; then, again here. I'd strreamline, if possible.